cpu/esp_common/Makefile.include

```diff
@@ -91,7 +97,10 @@ LINKFLAGS += -nostdlib -Wl,-gc-sections -Wl,-static
 ifeq (,$(filter esp_idf_heap,$(USEMODULE)))
   # use the wrapper functions for calloc to add correct overflow detection missing
   # in the newlib's version.
-  LINKFLAGS += -Wl,-wrap=_calloc_r
+  LINKFLAGS += -Wl,-wrap,_malloc_r
```

Comment:
@gschorcht are you sure these are needed? Applications are failing to link now that these actually are being applied to the linker command. In current master I only see `-wrap,_malloc_r`. Removing the others prevents linking from failing.

Reply:
Definitely yes, if module `esp_idf_heap` is used, all newlib heap function have to be mapped to the heap functions of the ESP-IDF.

I will take a look later when I'm back in the office.